docs/utilities/middleware_factory.md

```diff
@@ -78,9 +78,9 @@ This makes use of an existing Tracer instance that you may have initialized anyw
         ...
     ```
 
-When executed, your middleware name will [appear in AWS X-Ray Trace details as](../core/tracer) `## middleware_name`.
+When executed, your middleware name will [appear in AWS X-Ray Trace details as](/core/tracer) `## middleware_name`.
 
-For advanced use cases, you can instantiate [Tracer](../core/tracer) inside your middleware, and add annotations as well as metadata for additional operational insights.
+For advanced use cases, you can instantiate [Tracer](/core/tracer) inside your middleware, and add annotations as well as metadata for additional operational insights.
```

Comment:
Note to self: Investigate whether this broke due to Mike versioning

Relative path for internal linking should work 

https://www.mkdocs.org/user-guide/writing-your-docs/

Reply:
Just confirmed it works as expected, however the original is missing `.md` extension hence why it doesn't work.

`(..core/tracer.md)`